source/jquery.flot.time.js

```diff
@@ -227,7 +324,14 @@ API.txt for details.
             (opts.minTickSize && opts.minTickSize[1] ===
             "quarter") ? specQuarters : specMonths;
 
-        var timeUnitSize = opts.timeBase === 'seconds' ? timeUnitSizeSeconds : timeUnitSizeMilliseconds;
+        var timeUnitSize;
+        if (opts.timeBase === 'seconds') {
+            timeUnitSize = timeUnitSizeSeconds;
+        } else if (opts.timeBase === 'milliseconds') {
+            timeUnitSize = timeUnitSizeMilliseconds;
+        } else {
```

Comment:
The else case should default to milliseconds which is what is was before this feature was introduced

Reply:
I agree, I don't know what I was thinking mixing up to what different 'if' blocks default to. Which brings up the question: Why is it that in options, the timeBase defaults to 'seconds' but everywhere else in the code the default is 'milliseconds'?